websites/C/Coub/presence.ts

```diff
@@ -32,29 +31,27 @@ function getQuery() {
 function capitalizeFirstLetter(string: string) {
 	return string.charAt(0).toUpperCase() + string.slice(1);
 }
+const youtubeUrlRegex =
+	/^((?:https?:)?\/\/)?((?:www|m)\.)?((?:youtube\.com|youtu.be))(\/(?:[\w-]+\?v=|embed\/|v\/)?)([\w-]+)(\S+)?$/gm;
 function matchYoutubeUrl(url: string): boolean {
-	return !!url.match(
-		/^((?:https?:)?\/\/)?((?:www|m)\.)?((?:youtube\.com|youtu.be))(\/(?:[\w-]+\?v=|embed\/|v\/)?)([\w-]+)(\S+)?$/gm
-	);
+	return youtubeUrlRegex.test(url);
```

Comment:
do you really need this function? The regex is already a constant, surely you can just call .test on it